lxc/utils.go

```diff
@@ -181,3 +182,25 @@ func usage(name string, args ...string) string {
 
 	return name + " " + args[0]
 }
+
+// instancesExist iterates over a list of instances (or snapshots) and checks that they exist.
+func instancesExist(resources []remoteResource) error {
+	for _, resource := range resources {
+		// Handle snapshots.
+		if shared.IsSnapshot(resource.name) {
+			parent, snap, _ := shared.InstanceGetParentAndSnapshotName(resource.name)
+
+			_, _, err := resource.server.GetInstanceSnapshot(parent, snap)
+			if err != nil {
+				return fmt.Errorf(i18n.G("Failed to validate '%s:%s': %w"), resource.remote, resource.name, err)
```

Comment:
This should probably be ``Failed to validate "%s:%s": %w`` for consistency with `%q`.